benchmarks/synchrobench/run.sh

```diff
@@ -2,8 +2,17 @@
 
 dir=`pwd`
 
-# Get the output path as the first parameter. Default is <pwd>/output.
-output=${1:-${dir}/output}
+# trap ctrl-c and call ctrl_c()
```

Comment:
Sometimes it is needed to quit an experiments midway because we discovered an error early on.
This is a useful technique to allow exiting the experiments in the middle using a single CTRL+C stroke.
Without this piece of code, when the user press CTRL+C it only exits the current Java process, but continue the script to the next iteration. 

**How it works:**
`trap ctrl_c INT` means that the method `ctrc_c()` will be executed once the user presses CTRL+C.
`function ctrl_c()` simply updates a flag, which is checked before starting a new iteration (lines 118-121).

Reply:
OK